src/core/operator_list.js

```diff
@@ -226,10 +226,10 @@ var QueueOptimizer = (function QueueOptimizerClosure() {
         return i - ((i - iFirstSave) % 4);
       }
 
-      var q;
-      var isSameImage = false;
-      var iTransform, transformArgs;
-      var firstPIMXOArg0 = argsArray[iFirstPIMXO][0];
+      let q;
```

Comment:
*Optional nit:* I believe that you can simply inline this definition in the loop headers below, since it doesn't seem to ever be accessed outside of the loops.